cnd/src/http_api/protocol.rs

```diff
@@ -84,74 +94,129 @@ pub trait BetaParams {
 
 #[derive(Debug, Serialize)]
 pub struct LedgerEvents {
-    /// Keys are on of: "init", "deploy", "fund", "redeem", "refund".
-    /// Values are transactions.
-    transactions: HashMap<String, String>,
+    events: HashMap<ActionName, String>,
     status: EscrowStatus,
 }
 
 impl LedgerEvents {
-    fn new(status: EscrowStatus) -> Self {
-        Self {
-            transactions: HashMap::new(), /* if we want transaction here, we should save the
-                                           * events to the DB */
```

Comment:
There was the reason you mentioned in the PR description :)

I think we originally didn't have these here due to the `state` being different but that seems to have fixed itself up again :)